bin/protonPack

```diff
@@ -31,16 +35,16 @@ async function main() {
     // Run the dev server, but first find an available port
     if (isDevServer) {
         const server = require('../cli/server');
-        const port = await findPort();
+        const port = specifiedPort || await findPort();
```

Comment:
We can already use a custom port via NODE_ENV_PORT, to get the custom port via an arg, extract the arg inside `findPort`.

Reply:
idem for the publicPath add a new fn/.